src/release/breaking-changes/dispose.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Added missing `dispose()` for some disposable objects in Flutter
+description: >
+  'dispose()' might fail because of double disposal.
+---
+
+## Summary
+
+Missing calls to 'dispose()' are added for some disposable objects.
```

Comment:
For added context it would be nice to list somewhere in this doc examples of where we added previously missing disposes, e.g. "The FooController owned by the BarWidget was missing a dispose" or something like that.